lib/IMAP/FolderMapper.php

```diff
@@ -55,7 +55,7 @@ class FolderMapper {
 	 */
 	public function getFolders(Account $account, Horde_Imap_Client_Socket $client,
 							   string $pattern = '*'): array {
-		$mailboxes = $client->listMailboxes($pattern, Horde_Imap_Client::MBOX_SUBSCRIBED_EXISTS, [
+		$mailboxes = $client->listMailboxes($pattern, Horde_Imap_Client::MBOX_ALL_SUBSCRIBED, [
```

Comment:
@miaulalala I got myself confused as well but *all subscribed* doesn't mean we only get the subscribed mailboxes here. It actually returns *all*.

> Return all mailboxes regardless of subscription status, and ensure
>     *     the '\subscribed' attribute is set if mailbox is subscribed
>     *     (implies 'attributes' option is true).

for all subscribed

> Return subscribed mailboxes that exist on the server.

for subscribed exists